modules/sdk-coin-icp/src/lib/iface.ts

```diff
@@ -0,0 +1,95 @@
+import { TransactionType } from '@bitgo/sdk-core';
+
+export const RequestType = {
```

Comment:
For better type safety,
```suggestion
export enum RequestType = {
```

Reply:
changed